src/main/java/com/wdtinc/mapbox_vector_tile/build/MvtLayerProps.java

```diff
@@ -54,7 +62,16 @@ public int addValue(Object value) {
         }
 
         int nextIndex = vals.size();
-        final Integer mapIndex = vals.putIfAbsent(value, nextIndex);
+
+        // Android API 24 call
+        // final Integer mapIndex = vals.putIfAbsent(value, nextIndex);
```

Comment:
Let's make a 'putIfAbsent' utility function with the comment that it's to support android legacy.